sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/CopyModel.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer;
+
+import com.azure.ai.formrecognizer.models.CopyAuthorization;
+import com.azure.ai.formrecognizer.models.CustomFormModelInfo;
+import com.azure.ai.formrecognizer.models.OperationResult;
+import com.azure.ai.formrecognizer.training.FormTrainingClient;
+import com.azure.ai.formrecognizer.training.FormTrainingClientBuilder;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.util.polling.SyncPoller;
+
+/**
+ * Sample for copying a custom model from a source Form Recognizer resource to a target Form Recognizer resource.
+ */
+public class CopyModel {
+
+    /**
+     * Main method to invoke this demo.
+     *
+     * @param args Unused. Arguments to the program.
+     */
+    public static void main(final String[] args) {
+        // Instantiate a client that will be used to call the service.
+
+        FormTrainingClient client = new FormTrainingClientBuilder()
+            .credential(new AzureKeyCredential("{key}"))
+            .endpoint("https://{endpoint}.cognitiveservices.azure.com/")
+            .buildClient();
+
+        String targetResourceId = "target-resource-Id";
+        String targetResourceRegion = "target-resource-region";
+        final String copyModelId = "copy-model-Id";
```

Comment:
why is this final and the rest are not

Reply:
I will update this to not be final, it doesn't add a lot and will avoid too much of code in the user-samples.